monkey/monkey_island/cc/models/machine.py

```diff
@@ -12,12 +12,25 @@
 
 
 class Machine(MutableInfectionMonkeyBaseModel):
+    """Represents machines, VMs, or other network nodes discovered by Infection Monkey"""
+
     id: MachineID = Field(..., allow_mutation=False)
```

Comment:
Instead of type alias, this should just probably be a string. Mongodb by default generates hexadecimal ID's that are non-sequential. Even if we move to a database which generates non-sequential integers, they can still be cast to string.

Reply:
It's currently an `int`. Switching to a `str` would mean changing it in just one place, the `TypeAlias` definition. If I remove the `TypeAlias`, I need to change this everywhere it's used.

I'll consider switching to `MachineID` to `str`, but we should keep `MachineID`.